### ISSUE TITLE
Add validation analysis to the train loop. Implement train/valid split

### ISSUE THREAD

**Issue by edmondywen** [uclaacm/quora-nlp#5]
- add code to dataset class that creates a train/valid split based on some hyperparameter ratio (L + ratio)
- initialize validation dataset in main 
- use validation dataset and add validation accuracy calculation to train.py
